modules/rosa-getting-support.adoc

```diff
@@ -0,0 +1,25 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// support/rosa-getting-support.adoc
+
+[id="rosa-getting-support_{context}"]
+= Getting support for {product-title}
+
+[role="_abstract"]
+Learn about the Support options for {product-title}.
+
+If you experience difficulty with a procedure described in this documentation, visit the link:https://access.redhat.com/[Red Hat Customer Portal]. Through the Customer Portal, you can:
+
+* Search or browse through the Red Hat Knowledgebase of technical support articles about Red Hat products.
+* Access other product documentation.
+* Submit a support case to Red Hat Support:
+.. Click *Open a New Case*.
+.. Select the reason for the support ticket, such as `Defect/Bug` or `Account/Customer Service Request`.
+.. In the `Product` field, enter `OpenShift` to filter the list. Select `{product-title}` and the version from the drop-down menus.
+.. Complete the remaining fields.
+.. On the _Review_ page, select the correct cluster ID that you are contacting support about, and click `Submit`.
+
+You can also get support from link:https://aws.amazon.com/premiumsupport/[AWS Support].
+
+If you have a suggestion for improving this documentation or have found an error, submit a link:https://bugzilla.redhat.com/index.cgi[Bugzilla] report against the OpenShift Container Platform product for the Documentation component. Please provide specific details, such as the section name and {product-title} version.
```

Comment:
We generally avoid "please" in our docs. I think it's a minor thing in this case because it's directing the customer to submit a bug, which seems like a notable exception. But you could also say, "Be sure to provide..." ¯\_(ツ)_/¯

Reply:
Good catch, thanks!